doc/user-guide/managing-apps.md

```diff
@@ -1,7 +1,9 @@
 # Managing applications
 
 ## Labels
-
+```{tip}
+You can add labels to your app through the [CLI](./cli.md#add-labels)
```

Comment:
This can be added as a note at the end of the `Labels` section

Reply:
https://github.com/ploomber/doc/pull/261/commits/636f10183950535c600456653da2986448f17357